phoenix-core/src/it/java/org/apache/phoenix/end2end/BaseViewIT.java

```diff
@@ -130,7 +130,7 @@ protected void testUpdatableViewIndex(Integer saltBuckets, boolean localIndex) t
         rs = conn.createStatement().executeQuery("EXPLAIN " + query);
         String queryPlan = QueryUtil.getExplainPlan(rs);
         if (localIndex) {
-            assertEquals("CLIENT PARALLEL 3-WAY RANGE SCAN OVER _LOCAL_IDX_T [-32768,51]\n"
+            assertEquals("CLIENT PARALLEL "+ (saltBuckets == null ? 1 : saltBuckets)  +"-WAY RANGE SCAN OVER _LOCAL_IDX_T [-32768,51]\n"
```

Comment:
Added this so that I could add testNonSaltedUpdatableViewWithLocalIndex to ViewIT